docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,91 @@
+This guide was developed using a `Nokia G-240W-B` router (software version 3FE47969AGCA21). It should work for other models in the `G` series.
+
+!!! note
+There is no single way to do it right. Choose the one best suited for you.
```

Comment:
I think here is some indention missing, as the 'note' is only an empty box. See the preview: https://deploy-preview-791--pihole-docs.netlify.app/routers/nokia-G240WB/